W-19666248 HTTP Listener Updates (#1132)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/mule-gateway-autodiscovery-overview.adoc

endif::[]
:page-aliases: api-manager::api-auto-discovery-new-concept.adoc, api-manager::configure-auto-discovery-proxy-task.adoc

Configuring autodiscovery allows a deployed Mule runtime engine (Mule) application to connect with API Manager to download and manage policies and to generate analytics data. Additionally, with autodiscovery, you can configure your Mule applications to act as their own API proxy.

When autodiscovery is correctly configured in your Mule application, you can say that your application's API is _tracked by_ (green dot) or _paired to_ API Manager. You can associate an API in a Mule setup with only one autodiscovery instance at a given time.

[NOTE]
====

> The flow referenced for autodiscovery must use an xref:connectors::http/http-listener-ref.adoc[HTTP Listener] component. Other connectors that use HTTP as an underlying transport are not supported for policy enforcement.

Author: ```suggestion
The flow referenced for autodiscovery must use an xref:connectors::http/http-listener-ref.adoc[HTTP Listener] component. Other connectors that use HTTP as an underlying transport aren't supported for policy enforcement.
```
